test/ProgressOnderwijsUtils.Tests/TableValuedParameterTest.cs

```diff
@@ -18,6 +20,45 @@ public sealed class TableValuedParameterTest : TransactedLocalConnection
     {
         public enum SomeEnum { }
 
+        public struct CustomBlaStruct : IMetaObjectPropertyConvertible<CustomBlaStruct, string, CustomBlaStruct.CustomBlaStructConverter>
+        {
+            public struct CustomBlaStructConverter : IConverterSource<CustomBlaStruct, string>
+            {
+                public ValueConverter<CustomBlaStruct, string> GetValueConverter()
+                    => this.DefineConverter(v => v.CustomBlaString, v => new CustomBlaStruct(v));
+            }
+
+            public CustomBlaStruct(string value)
+                => CustomBlaString = value;
+
+            public string CustomBlaString { get; }
+
+            [MetaObjectPropertyLoader]
+            public static CustomBlaStruct MethodWithIrrelevantName(string value)
+                => new CustomBlaStruct(value);
+        }
+
+        [Fact]
+        public void ConvertibleProperty()
+        {
+            var value = SQL($@"select {new CustomBlaStruct("aap")}").ReadScalar<CustomBlaStruct>(Context);
+            PAssert.That(() => value.CustomBlaString == "aap");
+        }
+
+        [Fact]
+        public void ConvertibleNullablePropertyWitValue()
+        {
+            var value = SQL($@"select {new CustomBlaStruct("aap")}").ReadScalar<CustomBlaStruct?>(Context);
+            PAssert.That(() => value.Value.CustomBlaString == "aap");
+        }
+
+        [Fact]
+        public void ConvertibleNullablePropertyWithoutValue()
+        {
+            var value = SQL($@"select {default(CustomBlaStruct?)}").ReadScalar<CustomBlaStruct?>(Context);
+            PAssert.That(() => value == null);
+        }

```

Comment:
dus ik denk dat we hier ook extra tests willen voor null-containing struct, die volgens mij moet crashen.

Reply:
ik heb een test toegevoegd.